plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/rest/TestingLakekeeperCatalog.java

```diff
@@ -0,0 +1,214 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.iceberg.catalog.rest;
+
+import io.airlift.http.client.HttpClient;
+import io.airlift.http.client.Request;
+import io.airlift.http.client.StatusResponseHandler;
+import io.airlift.http.client.jetty.JettyHttpClient;
+import io.airlift.json.JsonCodec;
+import org.intellij.lang.annotations.Language;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.Network.NetworkImpl;
+
+import java.io.Closeable;
+import java.net.URI;
+import java.util.Map;
+
+import static io.airlift.http.client.StaticBodyGenerator.createStaticBodyGenerator;
+import static io.airlift.http.client.StatusResponseHandler.createStatusResponseHandler;
+import static io.airlift.http.client.JsonResponseHandler.createJsonResponseHandler;
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static java.util.Objects.requireNonNull;
+
+
+public final class TestingLakekeeperCatalog
+        implements Closeable {
+    public static final String DEFAULT_BUCKET = "examples";
+    public static final String DEFAULT_WAREHOUSE = "lakekeeper";
+    public static final String DEFAULT_PROJECT_ID = "00000000-0000-0000-0000-000000000000";
+    public static final String MINIO_ROOT_USER = "minio-root-user";
+    public static final String MINIO_ROOT_PASSWORD = "minio-root-password";
```

Comment:
Use same values as `io.trino.testing.containers.Minio`:
```suggestion
    public static final String MINIO_ROOT_USER = "accesskey";
    public static final String MINIO_ROOT_PASSWORD = "secretkey";
```